sys/clif/clif.c

```diff
@@ -168,22 +177,17 @@ ssize_t clif_add_attr(clif_attr_t *attr, char *buf, size_t maxlen)
     assert(attr);
     assert(attr->key);
 
-    /* if no length given, calculate it */
```

Comment:
I suggest we explicitly indicate in the documentation that `attr->key_len` should be set when calling this (a note is already in place for `attr->key`). Also please remove the note saying that the length is calculated.

Reply:
ack